src/css/base/typography.scss

```diff
@@ -7,6 +7,10 @@ h1, h2, h3, h4, h5, h6 {
   & + p {
     margin-top: 0.5rem;
   }
+
+  & + .push {
```

Comment:
Yea I want a different name to it because it's different then the standard push. It's a class that signifies that a div should be treated as a block of text.